bin/s2nc.c

```diff
@@ -13,39 +13,37 @@
  * permissions and limitations under the License.
  */
```

Comment:
Did you intend to change the file permissions on s2nc.c and s2nd.c?

Reply:
Yeah. Maybe it should be a separate PR.

But it was unclear to me why our C files where executable.

Reply:
That should be a separate PR, unless it's something clang-format does for you.